README.adoc

```diff
@@ -0,0 +1,1763 @@
+= The RSpec Style Guide
+:doctype: article
+
+This RSpec style guide outlines the recommended best practices for real-world
```

Comment:
I strong recommend not hard-wrapping paragraphs. That makes collaborative editing much more difficult.

Reply:
Can you please elaborate on this?

I can imagine that at several concurrent changes to the same paragraph scale the merge would become painful. But won't it be mostly manual if it was represented as a single line?

Reply:
I absolutely hate hard wrapping prose at a fixed column. Aside from constantly having to worry about reflow, the main issue is that you make one change at the start of the paragraph and every line in the paragraph gets shifted. This is terrible for collaborative editing. I advocate treating prose the way we do code. Each statement gets its own line. In prose, a statement is a sentence. That way, each change gets scoped to a single line. If you aren't going to use sentence-per-line (aka ventilated prose), the next best thing is soft wrapping.

Here's more context: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line